core/build/src/lib.rs

```diff
@@ -66,6 +66,7 @@ pub fn sgx_builder() -> Builder {
         .use_core()
         .ctypes_prefix("core::ffi")
         .allowlist_recursively(false)
+        .generate_comments(false)
```

Comment:
Why is this disabled again?

Reply:
brought it back
```
    Add back document generation to FFI crates
    
    Previously document generation was disabled during bindgen, to suppress
    a couple of warnings.  However disabling this prevents any documentation
    for the FFI crates. Since the header files are out of our control these
    warnings are ignored and documentation is generated.
```